eve/workers/openstack-multiple-nodes/requirements.sh

```diff
@@ -5,7 +5,7 @@ yum install -y epel-release
 PACKAGES=(
     curl
     git
-    python34-pip
+    python36-pip
```

Comment:
We also need to replace `pip3` with `pip34` below

Reply:
pip34 or pip36 ?

Reply:
Actually, `pip3.6` (oops)